client/src/main/java/org/asynchttpclient/netty/channel/DefaultChannelPool.java

```diff
@@ -356,6 +357,16 @@ public void flushPartitions(ChannelPoolPartitionSelector selector) {
         }
     }
 
+    @Override
+    public long getIdleChannelCount() {
+        return partitions.reduceValuesToLong(
```

Comment:
Yeah, I'd go with summing `ConcurrentLinkedDeque::size`. Those stats would just be an estimate anyway, you can't get a proper snapshot without freezing, which we of course don't want.